runtime/compiler/control/HookedByTheJit.cpp

```diff
@@ -7803,6 +7806,7 @@ void printIprofilerStats(TR::Options *options, J9JITConfig * jitConfig, TR_IProf
          iProfiler->printAllocationReport();
       if (TEST_verbose || options->getOption(TR_VerboseInterpreterProfiling))
          iProfiler->outputStats();
+      // iProfiler->traverseIProfilerTableAndGenerateHistograms(jitConfig);
```

Comment:
Does this need to be in this PR?

Reply:
Yes, I'd prefer to have the line there to serve as an example on how to generate the historgrams, should we want to.